src/css/styles.css

```diff
@@ -55,19 +55,42 @@ body {
 
 /*About us*/
 
-.title {
+#about-us-title {
     text-align: center;
     font-family: 'Courier New', Courier, monospace;
 }
```

Comment:
This info class is used on index.html, so I don't think you should change the font and also change it to a id.

Reply:
done